packages/react-scripts/config/config.js

```diff
@@ -0,0 +1,81 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const fs = require('fs');
+const chalk = require('chalk');
+const paths = require('./paths');
+
+/**
+ * Get the baseUrl of a compilerOptions object.
+ *
+ * @param {Object} options
+ */
+function getBaseUrl(options = {}) {
+  const baseUrl = options.baseUrl;
+
+  if (!baseUrl) {
+    return null;
+  }
+
+  if (baseUrl !== 'src' && baseUrl !== './src') {
+    console.error(
+      chalk.red.bold(
+        "You tried to set baseUrl to anything other than 'src'. This is not supported in create-react-app and will be ignored."
+      )
+    );
+
+    return null;
+  }
+
+  return 'src';
```

Comment:
Returning `src` regardless of user choice. What if user chose `./src` instead?

Reply:
`src` is the same as `./src` but by always returning `src` we know what we are working with when passing the value to webpack/jest.